shell/platform/windows/uwp_game_pad.cc

```diff
@@ -0,0 +1,223 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
```

Comment:
Can we split all the gamepad code out into a follow-up PR? It seems easily separable, and there's a lot of code in this PR at the moment.

Reply:
Yes, i'll split it out but will wait until all the feedback has been addressed so i can pull out the cleaned up version.

Reply:
(meaning the feedback for the rest of the change)

Reply:
gamepad code removed in next commit